_posts/2019-10-05-KrishnaGaur.md

```diff
@@ -0,0 +1,8 @@
+name: "Krishna Gaur"
+github_id: "gaurKrishna"
+
+
+Welcome to "Introduction to Git and Hactoberfest Study Jam!".
```

Comment:
Please make sure this is only 1-2 lines, 10-20 words max. Also remove `_posts/2019-10-05-KrishnaGaur` file using `git rm _posts/2019-10-05-KrishnaGaur`